gibbon-rts/Makefile

```diff
@@ -137,8 +137,8 @@ $(BUILD_DIR)/%.o: $(C_RTS_DIR)/%.o
 	mkdir -p $(BUILD_DIR) && \
 	mv $^ $@
 
-$(C_RTS_DIR)/%.o: $(C_RTS_DIR)/%.c
-	$(CC) $(CFLAGS) -c -o $@ $^
+$(C_RTS_DIR)/(NAME).o:
```

Comment:
```suggestion
# NOTE: we recompile without checking the timestamp of the source .c-file because
#       otherwise we're missing recompilations due to changes in the Make variables,
#       see https://github.com/iu-parfunc/gibbon/issues/244
$(C_RTS_DIR)/(NAME).o:
```

Reply:
Looks like we posted a comment simultaneously :D 

I noticed a missing "$" around `(NAME)` while I was building with this patch.